Jun-30 14:11:06.141 [main] DEBUG nextflow.cli.Launcher - $> nextflow run ../pipeline.nf --INPUT ./ --OUTDIR kms/ -resume -with-docker 'ubuntu:18.04'
Jun-30 14:11:06.333 [main] INFO  nextflow.cli.CmdRun - N E X T F L O W  ~  version 20.04.1
Jun-30 14:11:06.355 [main] INFO  nextflow.cli.CmdRun - Launching `../pipeline.nf` [amazing_davinci] - revision: e0301af02d
Jun-30 14:11:06.457 [main] DEBUG nextflow.config.ConfigBuilder - Enabling execution in Docker container as requested by cli option `-with-docker ubuntu:18.04`
Jun-30 14:11:06.554 [main] DEBUG nextflow.Session - Session uuid: 2576ffeb-d01a-40ad-a543-9088876cd271
Jun-30 14:11:06.555 [main] DEBUG nextflow.Session - Run name: amazing_davinci
Jun-30 14:11:06.557 [main] DEBUG nextflow.Session - Executor pool size: 24
Jun-30 14:11:06.631 [main] DEBUG nextflow.cli.CmdRun - 
  Version: 20.04.1 build 5335
  Created: 03-05-2020 19:37 UTC (12:37 PDT)
  System: Mac OS X 10.13.6
  Runtime: Groovy 2.5.8 on Java HotSpot(TM) 64-Bit Server VM 9.0.1+11
  Encoding: UTF-8 (UTF-8)
  Process: 56722@UWvirongs-share.local [10.18.239.7]
  CPUs: 24 - Mem: 32 GB (813.1 MB) - Swap: 1 GB (1022.2 MB)
Jun-30 14:11:06.677 [main] DEBUG nextflow.Session - Work-dir: /Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/testing/work [Mac OS X]
Jun-30 14:11:06.678 [main] DEBUG nextflow.Session - Script base path does not exist or is not a directory: /Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/bin
Jun-30 14:11:06.717 [main] DEBUG nextflow.Session - Observer factory: TowerFactory
Jun-30 14:11:06.722 [main] DEBUG nextflow.Session - Observer factory: DefaultObserverFactory
Jun-30 14:11:07.051 [main] DEBUG nextflow.Session - Session start invoked
Jun-30 14:11:07.820 [main] DEBUG nextflow.script.ScriptRunner - > Launching execution
Jun-30 14:11:07.839 [main] DEBUG nextflow.Session - Workflow process names [dsl1]: Aligning, Trimming
Jun-30 14:11:07.896 [PathVisitor-1] DEBUG nextflow.file.PathVisitor - files for syntax: glob; folder: ./; pattern: *_R{1,2}*.gz; options: [:]
Jun-30 14:11:08.144 [main] DEBUG nextflow.executor.ExecutorFactory - << taskConfig executor: null
Jun-30 14:11:08.144 [main] DEBUG nextflow.executor.ExecutorFactory - >> processorType: 'local'
Jun-30 14:11:08.153 [main] DEBUG nextflow.executor.Executor - [warm up] executor > local
Jun-30 14:11:08.161 [main] DEBUG n.processor.LocalPollingMonitor - Creating local task monitor for executor 'local' > cpus=24; memory=32 GB; capacity=24; pollInterval=100ms; dumpInterval=5m
Jun-30 14:11:08.334 [main] DEBUG nextflow.executor.ExecutorFactory - << taskConfig executor: null
Jun-30 14:11:08.334 [main] DEBUG nextflow.executor.ExecutorFactory - >> processorType: 'local'
Jun-30 14:11:08.342 [main] DEBUG nextflow.script.ScriptRunner - > Await termination 
Jun-30 14:11:08.343 [main] DEBUG nextflow.Session - Session await
Jun-30 14:11:08.632 [Actor Thread 3] INFO  nextflow.processor.TaskProcessor - [85/820081] Cached process > Trimming (1)
Jun-30 14:11:08.946 [Task submitter] DEBUG nextflow.executor.LocalTaskHandler - Launch cmd line: /bin/bash -ue .command.run
Jun-30 14:11:08.956 [Task submitter] INFO  nextflow.Session - [93/3203fc] Submitted process > Aligning (1)
Jun-30 14:11:09.837 [Task monitor] DEBUG n.processor.TaskPollingMonitor - Task completed > TaskHandler[id: 2; name: Aligning (1); status: COMPLETED; exit: 1; error: -; workDir: /Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/testing/work/93/3203fc770cefae152104e5fa62a853]
Jun-30 14:11:09.871 [Task monitor] ERROR nextflow.processor.TaskProcessor - Error executing process > 'Aligning (1)'

Caused by:
  Process `Aligning (1)` terminated with an error exit status (1)

Command executed:

  #!/bin/bash
  
  cat /Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/NC_045512.2.fasta
  #/usr/local/bin/bbmap.sh ref=/Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/NC_045512.2.fasta
  #/usr/local/bin/bbmap.sh in1=L001.R1.paired.fastq.gz in2=L001.R2.paired.fastq.gz out=L001.bam ref=/Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/NC_045512.2.fasta

Command exit status:
  1

Command output:
  (empty)

Command error:
  cat: can't open '/Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/NC_045512.2.fasta': No such file or directory

Work dir:
  /Users/uwvirongs/Documents/Michelle/covid_swift_pipeline/testing/work/93/3203fc770cefae152104e5fa62a853

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line
Jun-30 14:11:09.877 [main] DEBUG nextflow.Session - Session await > all process finished
Jun-30 14:11:09.881 [Task monitor] DEBUG nextflow.Session - Session aborted -- Cause: Process `Aligning (1)` terminated with an error exit status (1)
Jun-30 14:11:09.906 [main] DEBUG nextflow.Session - Session await > all barriers passed
Jun-30 14:11:09.932 [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow completed > WorkflowStats[succeededCount=0; failedCount=1; ignoredCount=0; cachedCount=1; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0; abortedCount=0; succeedDuration=0ms; failedDuration=787ms; cachedDuration=59.6s;loadCpus=0; loadMemory=0; peakRunning=1; peakCpus=1; peakMemory=0; ]
Jun-30 14:11:10.109 [main] DEBUG nextflow.CacheDB - Closing CacheDB done
Jun-30 14:11:10.127 [main] DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye
